### PR TITLE
Various improvements to Sidekiq documentation

### PR DESCRIPTION
**Pull request by ChrisBAshton**
- Made clearer the distinction between Sidekiq and govuk_sidekiq
- Describe how there is one shared Redis instance between machines
- Clarify Sidekiq Web port instructions
- Add an example Sidekiq console command for finding named workers

### CODE REVIEW

File: source/manual/sidekiq.html.md [hunk 1]
```diff
@@ -8,33 +8,65 @@ type: learn
...
 
 ## Monitoring
 
-There are three approaches for monitoring Sidekiq, via the [Sidekiq Web interface](#sidekiq-web),
-or the [Grafana dashboard](#sidekiq-grafana-dashboard) or the [Console](#sidekiq-from-the-console).
+There are three approaches for monitoring Sidekiq:
+
+- via the [Sidekiq Web interface](#sidekiq-web-aka-sidekiq-monitoring)
+- or the [Grafana dashboard](#sidekiq-grafana-dashboard)
+- or the [Console](#sidekiq-from-the-console)
```
Comment (AlanGabbianelli) [inline source/manual/sidekiq.html.md:43]:
```suggestion
- via the [Sidekiq Web interface](#sidekiq-web-aka-sidekiq-monitoring)
- via the [Grafana dashboard](#sidekiq-grafana-dashboard)
- via the [Console](#sidekiq-from-the-console)
```

File: source/manual/sidekiq.html.md [hunk 2]
```diff
@@ -8,33 +8,65 @@ type: learn
...
-multiple [Sidekiq] configurations used throughout [GOV.UK].
+that can display the current state of a Sidekiq installation. We have
+configured this as the [sidekiq-monitoring web app](https://github.com/alphagov/sidekiq-monitoring)
+to monitor multiple Sidekiq configurations used throughout GOV.UK.
 
-We have restricted public access as the Web UI allows modifying the state of
-[Sidekiq] queues. To access the Sidekiq monitoring UI for Production AWS,
-run:
+We have restricted public access because the Web UI allows modifying the
+state of [Sidekiq] queues. To access the Sidekiq monitoring UI for
```
Comment (AlanGabbianelli) [inline source/manual/sidekiq.html.md:54]:
```suggestion
state of Sidekiq queues. To access the Sidekiq monitoring UI for
```

Comment (ChrisBAshton) [reply source/manual/sidekiq.html.md]: Missed this one, thanks!

File: source/manual/sidekiq.html.md (hunk 1 above)
Comment (ChrisBAshton) [reply source/manual/sidekiq.html.md:43]: Personally I prefer what we have already, as it's clearer you're only expected to choose one of the options.

Comment (AlanGabbianelli) [reply source/manual/sidekiq.html.md:43]: If we say `There are three approaches for monitoring Sidekiq:`, it's clear that the list that follows is of those three approaches and that we're expected to choose one. I think the `or` works in a sentence (like it was before the change) but is unnecessary in a bullet point list.
Not a blocker though.

**Review by AlanGabbianelli — APPROVED** [review]

File: source/manual/sidekiq.html.md (hunk 1 above)
Comment (ChrisBAshton) [reply source/manual/sidekiq.html.md]: Actually on second read, I agree! I read your comment from the 'landing page' of the PR, which only shows the suggestion, and not the sentences that precedes it. Now I prefer the `or`.

Have overwritten my commit 👍

**Review by AlanGabbianelli — APPROVED** [review]

**Pull request merged**